stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java

```diff
@@ -79,6 +79,10 @@ else if(this.fileName.startsWith("./..")) {
 				String normalizedPath = p1.normalize().toString();
 				if (normalizedPath.length() > 0) {
 					this.fileName = normalizedPath; 
+					if (PlatformFinder.isWindows()) {
```

Comment:
I'm confused as https://github.com/AdoptOpenJDK/stf/blob/91a16e371f0ce95f3d4b7d9f769368c17ea98a67/stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java#L40 has already done the change? Why it is needed to change again?

Reply:
the call to normalize() , i.e., String normalizedPath = p1.normalize().toString(); in DirectoryRef is messing changing path separators updated earlier back to Windows style ones. So, that's why the test was failing.